altered createDATIMExport and created createPAWExport

flopez-bao opened the pull request:
- altered createDATIMExport and created createPAWExport - this is to address divergence of PAW and DATIM exports
- added pertinent tests
- edited packForDATIM function documentation
- related to https://jira.pepfar.net/browse/DP-1198 and datapack import tickets

JordanBalesBAO commented on R/createPAWExport.R:
```diff
@@ -0,0 +1,46 @@
+createPAWExport <- function(d) {
+
+  if (d$info$tool == "Data Pack") {
+    if (d$info$cop_year == 2022) {
```
Lets nuke the 2022 stuff

flopez-bao commented:
removed 2022 blocks from datapack calls in `createDATIMExport` and `createPAWExport`. Also added a new function as a concept `createDATAExport` plus test which can act as one function to handle export based on the type.

JordanBalesBAO commented on R/createDATIMExport.R:
```diff
@@ -31,9 +22,18 @@ createDATIMExport <- function(d) {
 
+  # handle OPU
   if (d$info$tool %in% c("OPU Data Pack", "PSNUxIM")) {
     datim_export <-  d$datim$OPU
```
Any ideas on how this will impact OPU's?

flopez-bao replied:
this was already in the OG code, so there should be no change, i only added the comment

jason-p-pickering reviewed (approved):
Looks good to me!

flopez-bao commented on R/createDATAExport.R:
```diff
@@ -0,0 +1,94 @@
+#' @export
+#'
+#'
+createDATAExport <- function(d, export_type = NULL) {
```
For now we are using createPAWExport and createDATIMExport so removed this one for now.
